akka-sample-sharding-scala/killrweather-fog/src/main/scala/sample/killrweather/fog/Fog.scala

```diff
@@ -73,19 +73,4 @@ final case class FogSettings(
     staggerStartup: FiniteDuration,
     host: String,
     registrationTimeout: FiniteDuration,
-    sampleInterval: FiniteDuration) {
-
-  def findHttpPort(attempt: Int): Option[Int] = {
-    val ds: DatagramSocket = DatagramChannel.open().socket()
-    try {
-      ds.bind(new InetSocketAddress(host, attempt))
-      Some(attempt)
-    } catch {
-      case NonFatal(e) =>
-        ds.close()
-        println(s"Unable to bind to port $attempt for http server to send data: ${e.getMessage}")
-        None
-    } finally
-      ds.close()
-  }
-}
```

Comment:
I think this was a too sketchy way to detect if there is a HTTP server running, so changed to prior-knowledge instead.